test/JDBC/expected/sysname-upgrade-vu-prepare.out

```diff
@@ -0,0 +1,23 @@
+CREATE FUNCTION sysname_func_vu_prepare(@a sys.sysname)
+RETURNS int
+AS
+BEGIN
+	return 1;
+END
+GO
+
+CREATE VIEW sysname_view_vu_prepare AS
+SELECT sysname_func_vu_prepare(-1) as a
+GO
+
+CREATE VIEW sysname_view_vu_prepare2 AS
+SELECT is_member(-1)
+GO
+
+CREATE VIEW sysname_view_vu_prepare3 AS
+SELECT is_member(-5)
```

Comment:
Chose to use is_member since it also took a sysname argument, though it looks like this object does not error out.